apis/core/v1beta1/openstackcontrolplane_webhook.go

```diff
@@ -362,6 +364,20 @@ func (r *OpenStackControlPlane) ValidateCreateServices(basePath *field.Path) (ad
 				maps.Keys(*r.Spec.Rabbitmq.Templates),
 				memcachedv1.CrMaxLengthCorrection) // omit issue with statefulset pod label "controller-revision-hash": "<statefulset_name>-<hash>"
 			errors = append(errors, err...)
+
+			// When a TopologyRef CR is referenced, fail if a different Namespace is
+			// referenced because is not supported
+			for rabbitmqName, rabbitmqSpec := range *r.Spec.Rabbitmq.Templates {
+				if rabbitmqSpec.TopologyRef != nil {
+					if err := topologyv1.ValidateTopologyNamespace(
+						rabbitmqSpec.TopologyRef.Namespace,
+						*basePath.Child("rabbitmq").Child("templates").Child(rabbitmqName),
+						r.Namespace,
+					); err != nil {
+						errors = append(errors, err)
+					}
+				}
```

Comment:
couldn't we just do `errors = append(errors, rabbitmqSpec.ValidateCreate(basePath.Child("rabbitmq").Child("template").Key(rabbitmqName), r.Namespace)...)` ?

Reply:
same you do bellow in the update

Reply:
indeed, just forgot to update this after adding ValidateCreate